index.bs

```diff
@@ -2629,7 +2629,7 @@ credential.
         Note: the [=user handle=] should not be a constant value across different accounts, even for [=non-discoverable credentials=], because some authenticators always create [=discoverable credentials=]. Thus a constant [=user handle=] would prevent a user from using such an authenticator with more than one account at the [=[RP]=]. Additionally,  a [=user handle=] having an empty value is known to be problematic in practice so is a bad choice, even if a constant value is desired.
 
     :   <dfn>displayName</dfn>
-    ::  A [=human palatability|human-palatable=] name for the user account, intended only for display. For example, "Alex P. Müller" or "田中 倫". The
+    ::  A [=human palatability|human-palatable=] name for the user account, intended only for display. For example, "Alex P. Müller" or "田中倫". The
```

Comment:
@aphillips suggested dropping the middle initial, which would apply to [line 2568](https://github.com/w3c/webauthn/pull/1528/files#diff-5e793325cd2bfc452e268a4aa2f02b4024dd9584bd1db3c2595f61f1ecf7b985R2568) as well.
```suggestion
    ::  A [=human palatability|human-palatable=] name for the user account, intended only for display. For example, "Alex Müller" or "田中倫". The
```